python/lsst/meas/extensions/scarlet/io.py

```diff
@@ -356,6 +358,10 @@ def parse_obj(cls, data: dict) -> "ScarletModelData":
             int(id): ScarletBlendData.fromDict(blend)
             for id, blend in data['blends'].items()
         }
+        if "filters" in dataShallowCopy:
+            # Support the original version,
+            # which used "filters" instead of the now canonical "bands."
+            dataShallowCopy["bands"] = dataShallowCopy.pop("filters")
```

Comment:
I presume the pop is deliberate but you could just leave both keys for backwards compatibility if needed.